pages/storage.vue

```diff
@@ -84,7 +84,7 @@
       </p>
     </div>
     <div class="is-flex is-flex-wrap-wrap is-justify-content-center mb-6">
-      <a href="https://brown.co1.qualtrics.com/jfe/form/SV_a5DbCjYNMb5iU0B">
+      <a href="https://brown.atlassian.net/servicedesk/customer/portal/16">
```

Comment:
Updating link to go directly to the request page

```suggestion
      <a href="https://brown.atlassian.net/servicedesk/customer/portal/16/group/55/create/218">
```